Turn commas into semicolons when the cell has many values

octopusinvitro opened the pull request:
We did this because the phone numbers sometimes had many
values for the same members and we want each phone in a
separate field in the Popolo file

Closes https://github.com/everypolitician/everypolitician-data/issues/13012

tmtmtmtm commented on scraper.rb:
```diff
@@ -24,6 +24,10 @@ def cell(profile, th)
   td.text.strip
 end
 
+def multiple_values(values)
```
The name of this function isn't particularly clear. (Or, from the other side, faced with a function called this, this isn't the implementation I'd expect to see in it)

tmtmtmtm commented on scraper.rb:
```diff
@@ -51,6 +55,7 @@ def scrape_mp(page)
  }
  data[:party_id] = data[:party].gsub(/\W+/,'').downcase
  data[:image] &&= URI.join(page, data[:image].text.gsub(' ','%20s')).to_s
+ data[:phone] = multiple_values(data[:phone])
```
It seems a little strange to do this as a separate step here, rather than in-place when the value is being extracted.

tmtmtmtm reviewed (changes requested):
I think you also want to make the whitespace changes as a separate commit.

octopusinvitro committed:
Turn commas into semicolons when the cell has many values

We did this because the phone numbers sometimes had many
values for the same members and we want each phone in a
separate field in the Popolo file

octopusinvitro committed:
Remove extra unneeded spaces
